sdk/servicebus/service-bus/src/serviceBusMessage.ts

```diff
@@ -817,9 +817,21 @@ export class ServiceBusMessage implements ReceivedMessage {
       this.messageId
     );
     if (this._context.requestResponseLockedMessages.has(this.lockToken!)) {
+      let receiverName;
+      if (this._context.batchingReceiver) {
+        receiverName = this._context.batchingReceiver.name;
+      } else if (this._context.streamingReceiver) {
+        receiverName = this._context.streamingReceiver.name;
+      }
+
+      if (this.sessionId) {
+        receiverName = this._context.messageSessions[this.sessionId].name;
+      }
```

Comment:
Any reason to not combine this with the previous if/else block? If the message is from sessions, why look for the non-session receivers?
Also, we found that empty string is a valid sessionId, so we can't use `if (this.sessionId)` check.

 I would suggest something like below:
```javascript
if (this.sessionId != undefined) {....}
else if (this._context.batchingReceiver) {....}
else if (this._context.streamingReceiver) {....}
```